x-pack/plugins/fleet/server/services/epm/packages/get_template_inputs.ts

```diff
@@ -57,6 +57,20 @@ export const templatePackagePolicyToFullInputStreams = (
   return fullInputsStreams;
 };
 
+export async function getTemplateInputs(
+  soClient: SavedObjectsClientContract,
+  pkgName: string,
+  pkgVersion: string,
+  format: 'yml',
+  prerelease?: boolean
+): Promise<string>;
+export async function getTemplateInputs(
+  soClient: SavedObjectsClientContract,
+  pkgName: string,
+  pkgVersion: string,
+  format: 'json',
+  prerelease?: boolean
+): Promise<{ inputs: FullAgentPolicyInputStream[] }>;
```

Comment:
I am not changing the return type here, simply overloading the function definition since the method will always return an object when format is 'json' and it will always return a string when format is 'yml'. 

This addition simply makes this method easier to consume. 

The `FullAgentPolicyInputStream[]` return type remains unchanged to the previous implementation and comes from the `templatePackagePolicyToFullInputStreams` function.